File: noduh/player_report.js
var PlayerReport = function() {}

PlayerReport.about = function (gameState) {
  console.log(gameState.players.length + " Players");
  console.log("---------");

  gameState.players.forEach (function (player) {
    console.log(player.name);
  });
}

module.exports = PlayerReport;

File: monolithic/player_report.cs
using System;

public class PlayerReport
{
  public static void about (GameState level)
  {
    Console.WriteLine (level.players.Count+" Players:");
    Console.WriteLine ("----------");

    foreach (Player player in level.players)
    {
      Console.WriteLine (player.Name);
    }
  }
}
